frontend/website/src/Testnet.re

```diff
@@ -118,15 +191,23 @@ module Styles = {
         display(`grid),
         color(Style.Colors.midnight),
         gridColumnGap(rem(1.5)),
-        gridTemplateColumns([rem(2.5), `auto, rem(6.), rem(2.5)]),
+        gridTemplateColumns([rem(1.), rem(5.5), rem(5.5), rem(2.5)]),
```

Comment:
now that this is a good chunk that's being duplicated, can we share the styles for header & rows?